File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/misc/Clock.java
package com.abubusoft.xenon.misc;

import android.os.SystemClock;

/**
 * Classe di utilità per il touchTimer
 * @author Francesco Benincasa
 *
 */
public abstract class Clock {
	/**
	 * Get time in millseconds, since device stopped.
	 * 
	 * @return
	 * 		time in millseconds
	 */
	public static long now()
	{
		return SystemClock.elapsedRealtime();
	}
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/misc/Clock.kt
package com.abubusoft.xenon.misc

import android.os.SystemClock

/**
 * Classe di utilità per il touchTimer
 * @author Francesco Benincasa
 */
object Clock {
    /**
     * Get time in millseconds, since device stopped.
     *
     * @return
     * time in millseconds
     */
    fun now(): Long {
        return SystemClock.elapsedRealtime()
    }
}